src/hooks/useFhirClient.test.tsx

```diff
@@ -536,8 +536,12 @@ describe('useDeleteResourceMutation', () => {
       });
     });
 
-    expect(axiosMock.history.delete[0].url).toBe(
-      `/v1/fhir/dstu3/Observation/${observationId}`,
-    );
+    await act(async () => {
```

Comment:
I might be able to peel some of these off but we had a surprising number of "runaway" tests, i.e., test ending before assertions were made and the advice out of jest was to wrap certain assertions with waitFor and also that waitFor may need to be wrapped with an act itself as it can cause state changes.

Reply:
```
     // Fails the test - attempted to import after jest has been torn down
      await waitFor(() => {
        expect(axiosMock.history.delete[0].url).toBe(
          `/v1/fhir/dstu3/Observation/${observationId}`,
        );
      });
      
       // Fails the test - attempted to import after jest has been torn down
      expect(axiosMock.history.delete[0].url).toBe(
        `/v1/fhir/dstu3/Observation/${observationId}`,
      );
      
      // Passes the test
      await act(async () => {
        await waitFor(() => {
          expect(axiosMock.history.delete[0].url).toBe(
            `/v1/fhir/dstu3/Observation/${observationId}`,
          );
        });
      });
      
      
      // Fails as expected
      await act(async () => {
        await waitFor(() => {
          expect(axiosMock.history.delete[0].url).toBe(
            `/v1/fhir/dstu3/Observati/${observationId}`,     // Alter assertion to be sure test CAN still fail
          );
        });
      });
```